File: src/day2/Zylinder.java
package day2;

public class Zylinder extends Kreis {

	private double hoehe;
	
	public Zylinder() {
		super();
	}
	
	public Zylinder (double radius, double hoehe) {
		super(radius);
		this.hoehe = hoehe;
	}
	
	public double getVolumen() {
		return this.getFlaeche() * this.hoehe;
	}
	
	public String toString() {
		return "Zylider mit Radius=" + this.getRadius() + ", Höhe=" + this.hoehe + ", Volumen=" + this.getVolumen() ; 
	}

	public double getHoehe() {
		return hoehe;
	}

	public void setHoehe(double hoehe) {
		this.hoehe = hoehe;
	}

}

File: lab/04-classes/zylinder.py
from kreis import Kreis

class Zylinder(Kreis):
    def __init__(self, radius, hoehe):
        super().__init__(radius)
        self.hoehe = hoehe

    def volumen(self):
        return self.flaeche() * self.hoehe

    def print(self):
        print("Zylinder mit Radius", self.radius, "und Höhe", self.hoehe)
        print("- Grundfläche = ", self.flaeche())
        print("- Volumen     = ", self.volumen())

if __name__ == "__main__":
    k = Zylinder(7, 10)
    k.print()